reana-test.yaml

```diff
@@ -0,0 +1,10 @@
+version: 0.3.0
```

Comment:
Remove `reana-test.yaml` since it can confuse people about its purpose

Reply:
Fixed in the year branches version.